users/tests.py

```diff
@@ -1,96 +1,86 @@
-        with self.assertRaises(ValueError):
-            with transaction.atomic():
-                db.objects.create_superuser(
-                    'email_2@address.com', 'first_name', 'last_name', '1900-01-01', 'password', is_superuser=False
-                )
-
-    def test_user(self):
-
-        db = get_user_model()
-
-        # Check if create_user creates a good user
-        user = db.objects.create_user(
-            'email@address.com', 'first_name', 'last_name', '1900-01-01', 'password'
-        )
-        self.assertEqual(user.email, 'email@address.com')
-        self.assertEqual(user.first_name, 'first_name')
-        self.assertEqual(user.last_name, 'last_name')
-        self.assertEqual(user.birth_date, '1900-01-01')
-        self.assertEqual(user.is_active, True)
-        self.assertEqual(user.is_staff, False)
-        self.assertEqual(user.is_superuser, False)
-        self.assertEqual(str(user), 'email@address.com')
-
-        # Check if email field is unique
-        with self.assertRaises(IntegrityError):
-            with transaction.atomic():
-                db.objects.create_user(
-                    'email@address.com', 'first_name', 'last_name', '1900-01-01', 'password'
-                )
-
-        # Check if create_user must have an email field
-        with self.assertRaises(ValueError):
-            with transaction.atomic():
-                db.objects.create_user(
-                    '', 'first_name', 'last_name', '1900-01-01', 'password'
-                )
-
-        # Check if create_user must have a first_name field
-        with self.assertRaises(ValueError):
-            with transaction.atomic():
-                db.objects.create_user(
-                    'email_1@address.com', '', 'last_name', '1900-01-01', 'password'
-                )
-
-        # Check if create_user must have a last_name field
-        with self.assertRaises(ValueError):
-            with transaction.atomic():
-                db.objects.create_user(
-                    'email_2@address.com', 'first_name', '', '1900-01-01', 'password'
-                )
-
-        # Check if create_user must have a birth_date field
-        with self.assertRaises(ValueError):
-            with transaction.atomic():
-                db.objects.create_user(
-                    'email_2@address.com', 'first_name', 'last_name', '', 'password'
-                )
+import pytest
+
+
+@pytest.fixture
+def superuser_model():
+    return get_user_model().objects.create_superuser(
+        'email@address.com', 'first_name', 'last_name', '1900-01-01', 'password'
+    )
+
+
+@pytest.mark.django_db
+def test_superuser_creation(superuser_model):
+    assert superuser_model.email == 'email@address.com'
+    assert superuser_model.first_name == 'first_name'
+    assert superuser_model.last_name == 'last_name'
+    assert superuser_model.birth_date == '1900-01-01'
+    assert superuser_model.is_active is True
```

Comment:
instead of asserting `superuser_model.is_active is True`, you can just `assert super_user_model.is_active`.
same for all the other boolean assertions.

Reply:
Done, waiting for your approval 💯